tests/test_interfaces.py

```diff
@@ -37,7 +37,12 @@ def test_interfaces(duthosts, enum_frontend_dut_hostname, tbinfo, enum_asic_inde
     verify_ip_address(host_facts, mg_facts['minigraph_lo_interfaces'])
 
     topo = tbinfo["topo"]["name"]
-    router_mac = duthost.facts['router_mac']
+    if tbinfo["topo"]["type"] == "t2":
```

Comment:
we can remove this check. should work for any topo

Reply:
@abdosi Updated based on your comments.